utility.py

```
@@ -942,7 +942,18 @@ def checkIfhasPlus(userid: int) -> bool:
     return True
 
 
-def missingLocalization(locale: str) -> None:
+async def missingLocalization(locale: str) -> None:
+    await run_blocking(_missingLocalization, locale)
```

Flake8 (7.3.0)

[error] undefined name 'run_blocking'

(F821)